fastapi_keycloak/api.py

```diff
@@ -423,6 +423,198 @@ def delete_role(self, role_name: str) -> dict:
             KeycloakError: If the resulting response is not a successful HTTP-Code (>299)
         """
         return self._admin_request(url=f'{self.roles_uri}/{role_name}', method=HTTPMethod.DELETE)
+        
+    @result_or_error(response_model=KeycloakGroup, is_list=True)
+    def get_all_groups(self) -> List[KeycloakGroup]:
+        """ Get all base groups of the Keycloak realm
+
+        Returns:
+            List[KeycloakGroup]: All base groups of the realm
+
+        Raises:
+            KeycloakError: If the resulting response is not a successful HTTP-Code (>299)
+        """
+        return self._admin_request(url=self.groups_uri, method=HTTPMethod.GET)     
+        
+    @result_or_error(response_model=KeycloakGroup, is_list=True)
+    def get_groups(self, group_names: List[str]) -> List[KeycloakGroup] or None:
+        """ Returns full entries of base Groups based on group names
+
+        Args:
+            group_names List[str]: Groups that should be looked up (names)
```

Comment:
The type for args must be in simple parentheses to match the google docstring and to be rendered correctly 
 
> `group_names (List[str]): ...`